org.gnome.Rhythmbox3.json

```diff
@@ -137,6 +139,71 @@
                 }
             ]
         },
+        /* gst-plugins-base needs cdparanoia to add support for cdda */
+        {
+            "disabled": true,
```

Comment:
Might as well remove it from the final commit.

Reply:
That's in error, obviously.